### PR TITLE
Get ready for the 0.13.0 release.

### PR DESCRIPTION
**Pull request by brendandburns**
cc @drubin @itowlson

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +103,33 @@ const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
...
+* `✓` Exactly the same features / API objects in both java-client and the Kubernetes
+  version.
+* `+` java-client has features or api objects that may not be present in the
+  Kubernetes cluster, but everything they have in common will work.
+* `-` The Kubernetes cluster has features the java-client library can't use
+  (additional API objects, etc).
+* `x` The Kubernetes cluster has no guarantees to support the API client of
+  this version, as it only promises _n_-2 version support. It is not tested,
+  and operations using API versions that have been deprecated and removed in
+  later server versions won't function correctly.
```
Comment (Flydiverny) [inline README.md:130]: java -> javascript

```suggestion
* `✓` Exactly the same features / API objects in both javascript-client and the Kubernetes
  version.
* `+` javascript-client has features or api objects that may not be present in the
  Kubernetes cluster, but everything they have in common will work.
* `-` The Kubernetes cluster has features the javascript-client library can't use
  (additional API objects, etc).
* `x` The Kubernetes cluster has no guarantees to support the API client of
  this version, as it only promises _n_-2 version support. It is not tested,
  and operations using API versions that have been deprecated and removed in
  later server versions won't function correctly.
```

**Comment by drubin** [comment]
Looks good with the suggested fixes. Tag me again and I will add the tag.

If I do it now it will get auto merged

File: README.md (hunk 1 above)
Comment (brendandburns) [reply README.md]: The perils of cut and paste! (thanks for the edits, applied)

**Comment by brendandburns** [comment]
@drubin comments addressed, please re-check.

Thanks!

**Pull request merged**